arrays-basics.js

```diff
@@ -2,46 +2,82 @@
  Task 8 (logArray): 
    Create a function logArray that receives an array as a parameter and logs each item in the array.
 **************************************************************/
-function logArray(arr) {}
 
-// logArray([1, 2, 3, 4, 5]);
+function logArray(arr) {
+  arr.forEach((value , index) => {console.log(value);})
+};
+
+logArray([1, 2, 3, 4, 5]);
 
 /**************************************************************
    Task 9 (logArrayWithIndex): 
      Create a function logArrayWithIndex that receives an array as a parameter and logs each item in the array along with its index.
   **************************************************************/
-function logArrayWithIndex(arr) {}
 
-// logArrayWithIndex(["apple", "banana", "orange"]);
+function logArrayWithIndex(arr) {
+  arr.forEach((value, index) => {
+    console.log(value, " ", index);
+  });
+}
+
+logArrayWithIndex(["apple", "banana", "orange"]);
 
 /**************************************************************
    Task 10 (logEvenNumbers): 
      Create a function logEvenNumbers that receives an array of numbers as a parameter and logs only the even numbers in the array.
   **************************************************************/
-function logEvenNumbers(arr) {}
 
-// logEvenNumbers([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]);
+function logEvenNumbers(arr) {
+  arr.forEach((value, index) => {
+    if (value % 2 == 0) {
+      console.log(value);
+    }
+  });
+}
+
+logEvenNumbers([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]);
 
 /**************************************************************
    Task 11 (logArrayBackwards): 
      Create a function logArrayBackwards that receives an array as a parameter and logs each item in the array in reverse order.
   **************************************************************/
-function logArrayBackwards(arr) {}
 
-// logArrayBackwards(["one", "two", "three", "four"]);
+function logArrayBackwards(arr) {
+  for (let i = arr.length - 1; i >= 0; i--) {
+    console.log(arr[i]);
+  }
+}
+
+logArrayBackwards(["one", "two", "three", "four"]);
 
 /**************************************************************
    Task 12 (logLastItem): 
      Create a function logLastItem that receives an array as a parameter and logs the last item in the array.
   **************************************************************/
-function logLastItem(arr) {}
 
-// logLastItem(["a", "b", "c", "d"]);
+ function logLastItem(arr) {
+  arr.map((value, index) => {
```

Comment:
You don't need to loop over every item inside of the array to get the last item, instead you can do this:
```
arrLen = arr.length;
console.log(arr[arrLen-1]
```